tokens/theme-semantics.json

```diff
@@ -1 +1,38 @@
-{}
\ No newline at end of file
+{
+  "color": {
+    "bg": {
+      "canvas": {
+        "value": "{color.theme.bg}",
```

Comment:
It's not a semantic token. Leonardo generates the color scales, and a global background color for each theme. Yesterday, while workin on the internal demo I realized we needed to capture that global theme background color that Leonardo generates. All color contrast ratios are calculated against that background color, and is the background color that we'll use as the UI canvas (e.g., `#fff` for Light theme).

Reply:
Why is it not a semantic token, it looks like one to me, but i'm sure i'm missing some nuance here?
And if that is defined only to satisfy leonardo why does it exist here? your tool that generates the tokens should prune it out?

Reply:
> And if that is defined only to satisfy leonardo why does it exist here? your tool that generates the tokens should prune it out?

It's not. We need to consume it.

Sorry for the confusion, I'll try to clarify things a bit more.

Leonardo's output is:

- color scales for the different hues and themes
- global background for each theme

We need to feed both to Tokens Studio, as both are generated based on the input parameters to Leonardo. What is more, the contrast ratios are calculated against that background color generated by Leonardo based on the input params.

Leonardo automatically generates the names for the colors (`blue900`), as well as for the theme background (`background`).

We transformed Leonardo color names to a format we chose, that is, we use `color.blue.900` instead of Leonardo's `blue900`). I just did a similar treatment for the generated theme background color, so we use `color.theme.bg` instead of Leonardo's `background`. So, it follows the same structure `color.` as the top-level namespace, `theme` instead of a specific hue, and then `bg` as a shorthand for background.

So, `color.theme.bg` is defined in the theme as a **base token** in each token set `theme-light`, `theme-dark`, etc.

Then, `color.bg.canvas` is a **semantic token** defined in `theme-semantics` that consumes the `color.theme.bg` base token.